auditlog/models/rule.py

```diff
@@ -566,10 +539,10 @@ def unsubscribe(self):
             if act_window:
                 value = 'ir.actions.act_window,%s' % act_window.id
                 act_window.unlink()
-                ir_value = ir_values_model.search(
+                ir_default = model_ir_default.search(
```

Comment:
Earlier on, ir.values was used for two purposes: registering default values, and exposing window actions under the More... button of certain record types. I believe that in this case, it was used for the latter purpose so after removing the window action of a model's logging you don't have to check for pending defaults so you can remove the modified lines.